cyw43/src/events.rs

```diff
@@ -1,5 +1,5 @@
 #![allow(dead_code)]
-#![allow(non_camel_case_types)]
+#![allow(non_camel_case_types, non_upper_case_globals)]
```

Comment:
ths is a false positive in `rust-analyzer`, we shouldn't workaround it in the repo.